src/ray/core_worker/core_worker.cc

```diff
@@ -3686,7 +3684,7 @@ void CoreWorker::HandleDirectActorCallArgWaitComplete(
   // Post on the task execution event loop since this may trigger the
   // execution of a task that is now ready to run.
   task_execution_service_.post(
-      [=] {
+      [request = std::move(request), this] {
```

Comment:
nit: no action needed, the general practice is to place `this` as the first captured variable, just for code style consistency.

Reply:
agree, that's my bad